serverless/functions/serverless-developing-go-functions.adoc

```diff
@@ -0,0 +1,35 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-developing-go-functions"]
+= Developing Golang functions
+:context: serverless-developing-go-functions
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+After you have xref:../../serverless/functions/serverless-functions-getting-started.adoc#serverless-create-func-kn_serverless-functions-getting-started[created a Golang function project], you can modify the template files provided to add business logic to your function.
+
+[id="prerequisites_serverless-developing-go-functions"]
+== Prerequisites
+
+* Before you can develop functions, you must complete the steps in xref:../../serverless/functions/serverless-functions-setup.adoc#serverless-functions-setup[Setting up {FunctionsProductName}].
+
+include::modules/serverless-go-template.adoc[leveloffset=+1]
+
+[id="serverless-developing-go-functions-about-invoking_{context}"]
+= About invoking Golang functions
```

Comment:
This should be a level 2 heading now that it's in the assembly (there should only ever be one level 1 heading in a file anyways). It's also causing the indentation for the rest of the TOC to be off I think.